Automattic/wp-calypso
# Redirect Jetpack Cloud /manage/pricing to Jetpack A4A landing page.

This PR redirects the Jetpack Cloud `/manage/pricing` page to the https://jetpack.com/for-agencies/ landing page.

PT: pbNhbs-axp-p2




## Proposed Changes

* Apply a redirect in the /manage/pricing controller to redirect to the Jetpack "For Agencies" landing page (https://jetpack.com/for-agencies/).

## Testing Instructions



- Checkout this branch and spin up calypso green, OR click the _Jetpack Cloud live (direct link)_ below. ⤵️
- Go to the `/manage/pricing` page.
- Verify you are immediately redirected to https://jetpack.com/for-agencies/ landing page.




## Pre-merge Checklist
